packages/treats/server/renderer.js

```diff
@@ -173,11 +173,25 @@ const renderer = async (req, res, routerContext, customRenderers) => {
         { template } = req.renderParams,
         flushedChunks = flushChunks(assetsStats, {
             chunkNames,
-            before: ["manifest", "vendor"],
+            before: ["manifest", "vendor", "service-worker"],
             after: ["main"]
         }),
-        { styles, js, cssHash: css } = flushedChunks;
-    const jsTags = js.toString(),
+        { styles, cssHash: css, scripts: jsFiles, publicPath } = flushedChunks,
+        swFilename = assetsStats.assetsByChunkName["service-worker"];
+
+    //Custom js files tag generator based on webpack-flush-chunks API to modify service worker script tag
+    const jsTagGenerator = files =>
+        files
+            .map(file => {
+                if (file === swFilename) {
+                    return `<script id="service-worker" type="text/javascript" src="${publicPath}/${file}"></script>`;
```

Comment:
I prefer to use data attributes like `data-treats-sw` or something similar rather than `id="service-worker"` since it would confuse the user if they want to have `id="service-worker"` in one of their component

Reply:
Tutorial & example are next steps. This pr will only consists of workbox implementation and addition in treats.
For the jQuery usage, I'll change it. Thanks for informing

Reply:
Alrighty, but please at least have an entry in the docs to explain on how people can add PWA support and how the `workbox` config in `treats.config.js` works 😄

Reply:
alright will do :+1: